docs/cookbook/using-docker.md

```diff
@@ -0,0 +1,343 @@
+# Building Docker images for Apostrophe projects
+
+[Docker](https://www.docker.com/) is a containerization platform that lets developers build an image for their projects and then run it anywhere. This document will guide you through setting up Docker and using it to create an image for hosting your Apostrophe project.
+
+The initial steps of this guide will assume that you will be hosting your database, project, and uploaded assets on the same server. The second part will outline steps for configuring to use the AWS S3 service for hosting assets. Finally, the third portion will provide guidance for using the MongoDB Atlas multi-cloud database.
```

Comment:
"This guide is for production, not development. If you want to use Docker as a development environment, you can explore using a persistent Docker volume for your project, but bear in mind that commands like `npm install` can be very slow in such a configuration."

Reply:
Added in place of last sentence in first paragraph.